pool/config.lua

```diff
@@ -9,6 +9,9 @@ box.once("init", function()
     box.schema.user.create('test', { password = 'test' })
     box.schema.user.grant('test', 'read,write,execute', 'universe')
 
+    box.schema.user.create('testNoExecute', { password = 'test' })
+    box.schema.user.grant('testNoExecute', 'read,write', 'universe')
```

Comment:
It looks a little bit more idiomatic.
```suggestion
    box.schema.user.create('test_noexec', { password = 'test' })
    box.schema.user.grant('test_noexec', 'read,write', 'universe')
```

Reply:
Updated.